CHANGELOG.md

```diff
@@ -4,6 +4,60 @@ All notable changes to this project will be documented in this file.
+- chore\(deps\): update dependency percona/mongodb\_exporter to v0.43.1 [\#860](https://github.com/voxpupuli/puppet-prometheus/pull/860) ([pccibot](https://github.com/pccibot))
+- chore\(deps\): update dependency oliver006/redis\_exporter to v1.68.0 [\#859](https://github.com/voxpupuli/puppet-prometheus/pull/859) ([pccibot](https://github.com/pccibot))
+- chore\(deps\): update dependency ncabatoff/process-exporter to v0.8.5 [\#858](https://github.com/voxpupuli/puppet-prometheus/pull/858) ([pccibot](https://github.com/pccibot))
+- chore\(deps\): update dependency galexrt/dellhw\_exporter to v1.13.13 [\#857](https://github.com/voxpupuli/puppet-prometheus/pull/857) ([pccibot](https://github.com/pccibot))
+- haproxy\_exporter: Move data from hiera to puppet class [\#845](https://github.com/voxpupuli/puppet-prometheus/pull/845) ([bastelfreak](https://github.com/bastelfreak))
+- process\_exporter: Move data from hiera to puppet class [\#844](https://github.com/voxpupuli/puppet-prometheus/pull/844) ([bastelfreak](https://github.com/bastelfreak))
+- pushgateway\_exporter: Move data from hiera to puppet class [\#843](https://github.com/voxpupuli/puppet-prometheus/pull/843) ([bastelfreak](https://github.com/bastelfreak))
+- mysql\_exporter: Move data from hiera to puppet class [\#842](https://github.com/voxpupuli/puppet-prometheus/pull/842) ([bastelfreak](https://github.com/bastelfreak))
+- node\_exporter: Move data from hiera to puppet class [\#841](https://github.com/voxpupuli/puppet-prometheus/pull/841) ([bastelfreak](https://github.com/bastelfreak))
+- beanstalkd\_exporter: Move data from hiera to puppet class [\#840](https://github.com/voxpupuli/puppet-prometheus/pull/840) ([bastelfreak](https://github.com/bastelfreak))
+- snmp\_exporter: Move data from hiera to puppet class [\#839](https://github.com/voxpupuli/puppet-prometheus/pull/839) ([bastelfreak](https://github.com/bastelfreak))
+- statsd\_exporter: Move data from Hiera to puppet class  [\#838](https://github.com/voxpupuli/puppet-prometheus/pull/838) ([bastelfreak](https://github.com/bastelfreak))
+- allow installation of rabbitmq\_exporter \>= 1.0.0 [\#835](https://github.com/voxpupuli/puppet-prometheus/pull/835) ([TheMeier](https://github.com/TheMeier))
+- remove custom use of archive for several exporters [\#834](https://github.com/voxpupuli/puppet-prometheus/pull/834) ([TheMeier](https://github.com/TheMeier))
+
+**Closed issues:**
+
+- rabbit\_mq exporter cannot be download for versions \> 1.0.0-RC8 [\#662](https://github.com/voxpupuli/puppet-prometheus/issues/662)
+- Split localstorage creation and localstorage configuration [\#632](https://github.com/voxpupuli/puppet-prometheus/issues/632)
+- Old URL releases are not removed [\#625](https://github.com/voxpupuli/puppet-prometheus/issues/625)
+- TODO: patch prometheus::daemon to support custom extract directories [\#606](https://github.com/voxpupuli/puppet-prometheus/issues/606)
```

Comment:
I added skip-changelog here, can you please regenerate it?

Reply:
done